src/config/index.js

```diff
@@ -247,6 +249,48 @@ config.settings.apiExpanders = [
   },
 ];
 
+if (prefixPath) {
+  const ControlPanelAsyncPropExtender = {
+    path: `${prefixPath}/controlpanel`,
```

Comment:
> Note: There's still an issue with asyncConnect Calls. I'm looking into it.

This fixes the issue. The async action dispatcher in https://github.com/plone/volto/blob/7b85e97c5e7c5046f1fce432bbbfd3f6842bd15f/src/components/manage/Controlpanels/Controlpanels.jsx#L311 doesn't seem to work well in ssr when prefixed paths. Do you see any other way of doing this without per route extenders?